samples/scriptable/bubble.html

```diff
@@ -86,13 +86,14 @@
 					},
 
 					hoverBorderWidth: function(context) {
-						return Math.round(8 * context.data.v / 1000);
+						var value = context.dataset.data[context.dataIndex];
+						return Math.round(8 * value.v / 1000);
 					},
 
 					radius: function(context) {
-						var data = context.data;
+						var value = context.dataset.data[context.dataIndex];
```

Comment:
@etimberg this is the way how to read the current value with the minimal context. What do you think?